packages/frint/src/App.ts

```diff
@@ -116,6 +123,15 @@ export class App {
       throw new Error('Must provide `name` in options');
     }
 
+    // expose methods as class properties
+    Object.keys(this.options.methods).forEach(methodName => {
+      const method = this.options.methods[methodName];
+
+      if (typeof method === 'function') {
+        this[methodName] = method.bind(this);
```

Comment:
it will be useful if we check if there is already an existing method with the same name first.

if it exists, we throw an Error.

Reply:
Good idea, will do 🙂